### PR TITLE
ST-510: Review fixes

### PR DESCRIPTION
**Pull request by ArtObr**

### CODE REVIEW

File: sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_commit_batch.py [hunk 1]
```diff
@@ -3,13 +3,13 @@
 from sovtokenfees.static_fee_req_handler import txn_root_serializer
 
 
-def test_token_batch_handler_commit_batch(token_batch_handler, utxo_cache):
+def test_token_batch_handler_commit_batch(utxo_batch_handler, utxo_cache):
```
Comment (ashcherbakov) [inline sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_commit_batch.py:6]: A correct test name is `test_utxo_batch_handler_commit_batch`

File: sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_post_batch_applied.py [hunk 2]
```diff
@@ -1,11 +1,11 @@
 from collections import namedtuple
 
 
-def test_token_batch_handler_post_batch_applied(token_batch_handler, utxo_cache):
+def test_token_batch_handler_post_batch_applied(utxo_batch_handler, utxo_cache):
```
Comment (ashcherbakov) [inline sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_post_batch_applied.py:4]: A correct test name is `test_utxo_batch_handler_post_batch_applied`

File: sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_post_batch_rejected.py [hunk 3]
```diff
@@ -3,12 +3,12 @@
 from sovtoken import TOKEN_LEDGER_ID
 
 
-def test_token_batch_handler_post_batch_rejected(token_batch_handler, utxo_cache):
+def test_token_batch_handler_post_batch_rejected(utxo_batch_handler, utxo_cache):
```
Comment (ashcherbakov) [inline sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_post_batch_rejected.py:6]: A correct test name is `test_utxo_batch_handler_post_batch_rejected`

File: sovtokenfees/sovtokenfees/main.py [hunk 4]
```diff
@@ -97,4 +97,26 @@ def filter_fees(ledger_id: int, txn: Any):
     node.master_replica.register_hook(ReplicaHooks.APPLY_PPR,
                                       three_pc_handler.check_recvd_pre_prepare)
 
+    # Pluggable request handlers initialization
+
+    # Firstly registered handlers must be txn main purposed
+    # node.write_request_manager.register_req_handler(SetFeesHandler(node.db_manager,
+    #                                                   node.write_req_validator))
+    # Replace XFER handler
+    # del node.write_request_manager.request_handlers[XFER_PUBLIC][-1]
```
Comment (ashcherbakov) [inline sovtokenfees/sovtokenfees/main.py:106]: `node.write_request_manager.remove_req_handlers` should be used instead

**Review by ashcherbakov — APPROVED** [review]

**Review by KitHat — APPROVED** [review]

File: sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_commit_batch.py (hunk 1 above)
Comment (ArtObr) [reply sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_commit_batch.py:6]: fixed

File: sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_post_batch_rejected.py (hunk 3 above)
Comment (ArtObr) [reply sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_post_batch_rejected.py:6]: fixed

File: sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_post_batch_applied.py (hunk 2 above)
Comment (ArtObr) [reply sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_post_batch_applied.py:4]: fixed

File: sovtokenfees/sovtokenfees/main.py (hunk 4 above)
Comment (ArtObr) [reply sovtokenfees/sovtokenfees/main.py:106]: fixed

**Pull request merged**